librz/analysis/block.c

```diff
@@ -839,34 +839,34 @@ static bool automerge_predecessor_successor_cb(ut64 addr, void *user) {
 		return true;
 	}
 	bool found;
-	RzAnalysisBlock *pred = ht_up_find (ctx->predecessors, (ut64)block, &found);
+	RzAnalysisBlock *pred = ht_up_find (ctx->predecessors, HT_PTR(block), &found);
```

Comment:
I think this is just wrong.. :/ Instead of using this new `HT_PTR` thing we should just use `ht_pp_find`, which has a `void *` as key and a `void *` as value. By doing so you don't need to convert between ut64 and void * .